### PR TITLE
Upgrade to ocamlformat.0.13.0

### PR DESCRIPTION
**Pull request by pascutto**

### CODE REVIEW

File: src/ppx_irmin/lib/deriver.ml [hunk 1]
```diff
@@ -103,8 +103,7 @@ module Located (A : Ast_builder.S) : S = struct
                       rec_detected := true;
                       generic_name
                       (* If not a base type, assume a composite generic with the
-                         same naming convention *)
-                      )
+                         same naming convention *) )
```
Comment (pascutto) [inline src/ppx_irmin/lib/deriver.ml]: This one does not look very nice and is reported in https://github.com/ocaml-ppx/ocamlformat/issues/1228.

**Comment by CraigFe** [comment]
`ocaml-ci` is failing because of an out-of-date opam-repository snapshot. https://github.com/ocurrent/ocaml-ci/issues/26

**Comment by pascutto** [comment]
> `ocaml-ci` is failing because of an out-of-date opam-repository snapshot. [ocurrent/ocaml-ci#26](https://github.com/ocurrent/ocaml-ci/issues/26)

We'll wait until the images are updated so the CI is still useful for the next PRs.

**Comment by CraigFe** [comment]
Travis CI failure unrelated.

**Pull request merged**